packages/zowe-explorer/src/utils/AuthUtils.ts

```diff
@@ -83,6 +83,7 @@ export class AuthUtils {
         const correlation = ErrorCorrelator.getInstance().correlateError(moreInfo?.apiType ?? ZoweExplorerApiType.All, errorDetails, {
             profileType: profile?.type,
             ...Object.keys(moreInfo).reduce((all, k) => (typeof moreInfo[k] === "string" ? { ...all, [k]: moreInfo[k] } : all), {}),
+            templateArgs: { ...(moreInfo.templateArgs ?? {}), profileName: profile?.name ?? "" },
```

Comment:
The template args are essentially key/value pairs of strings, and in that scenario you would want to pass the profileName as a template arg rather than the profile, as these template args are passed right into Handlebars. If you have access to the profile at the time of the errorHandling call, it should be passed into `moreInfo` as the `profile` property.

As for the `?? {}` check, it is needed because `moreInfo` may be undefined - that said I do see a missing `?` before `.templateArgs`, so I'll add that.